### ISSUE TITLE
专家轨迹加载时可能使用了错误坐标系下的根节点角速度

### ISSUE THREAD

**Issue by toonasinensis** [Roboparty/UFO#1]
在 load_expert_trajectories_from_motion_lib 中，代码直接从 motion library 获取根节点角速度，并将其拼接到专家状态中：

ref_body_angular_vels = motion_res["body_ang_vel_t"]

ref_ang_vel = ref_body_angular_vels[:, 0]

state = torch.cat(
    [
        ref_dof_pos,
        ref_dof_vel,
        projected_gravity,
        ref_ang_vel,
    ],
    dim=-1,
)

这里的 ref_ang_vel 可能仍然是世界坐标系下的角速度，但在线环境中策略使用的 base_ang_vel 通常是机器人本体坐标系下的角速度。

如果两者坐标系不同，会导致专家数据和在线环境观测不一致。

**Comment by Xuewang01** [comment]
Fixed in [92c9b6c](https://github.com/Roboparty/UFO/commit/92c9b6c5972c1cf83c5952ce62d6396c1ecce426).

This issue is covered by the same reference-observation fix as #2. Expert/reference `base_ang_vel` is now computed with the same coordinate convention as simulator observations:

```python
quat_rotate_inverse(base_quat, world_base_ang_vel, w_last=True) * obs_scales["base_ang_vel"]
